documentation/book/con-operator-configuration.adoc

```diff
@@ -0,0 +1,63 @@
+[id='operator-configuration-{context}']
+= Operator Configuration
+
+The operator itself can be configured through the following environment variables.
```

Comment:
This seems to be a duplicate content with `documentation/book/con-configure-operator.adoc`. I think you should keep this one and remove it from the other one?